.gitignore

```diff
@@ -28,3 +28,7 @@
 *.safetensors
 *.gguf
 *.vmfb
+e2eshark/onnx/operators/*/model.onnx
```

Comment:
This is fine, but you can also put a .gitignore in the `e2eshark` directory (or any of the other directories). See how I have a similar file here: https://github.com/ScottTodd/SHARK-TestSuite/blob/onnx-testsuite/iree_tests/.gitignore on https://github.com/nod-ai/SHARK-TestSuite/pull/68.

That would be more robust to directory moves. Consider if someone renames `e2eshark` (or moves it up/down a level) they would also need to update the path here in the root file. If there was a .gitignore in that folder itself, the rename would be kept local.